src/common/Validator.ts

```diff
@@ -201,7 +201,7 @@ export function validateV0ConfigData(data: ConfigV0) {
 function cleanURL(url: string): string {
     let updatedURL = url;
     if (updatedURL.includes('\\')) {
-        updatedURL = updatedURL.toLowerCase().replace(/\\/gi, '/');
```

Comment:
Are there any cases where later we compare the server url that it make break now if it's there any casing mistmatch?

Reply:
Shouldn't be. We were doing this to make sure the path name was lower case, and this function is mostly to get rid of double slashes. We do a lower case when we compared host and origin in other parts of the code.

Reply:
@devinbinnie isn't it this: https://github.com/mattermost/desktop/blob/master/src/common/utils/url.ts#L9 that we should be changing?